apps/docs/src/app/platform/component-docs/platform-forms/platform-textarea/platform-textarea-examples/platform-textarea-autogrow-example.component.html

```diff
@@ -0,0 +1,151 @@
+<main class="fd-page">
+    <div *ngIf="true" class="fd-page__content fd-has-background-color-neutral-2">
+        <fdp-form-group #fg4 [multiLayout]="true" [formGroup]="form">
+            <fdp-form-field
```

Comment:
fdp-form-help is not needed, we are showing the text area, not this component

Reply:
Do you mean `fdp-form-field`? We are showing textarea in the form-field's context most of the times because we want the end user to understand where things like placeholder, label, validation arrays etc need to go and how they work with `fdp-textarea` or any child component. When just provided with `fdp-textarea`, we are not showing how it would work with `fdp-form-field` as such, or how error messages are thrown. 
I see even in core textarea, we are showing every example in the context of `fd-form-item` or `fd-form-input-message-group` etc. 
I think this is required.